src/button-dropdown/__integ__/button-dropdown-items.test.ts

```diff
@@ -17,3 +17,21 @@ test(
     expect(await browser.getWindowHandles()).toHaveLength(2);
   })
 );
+
+test(
+  'focus on first menu item when dropdown is opened',
+  useBrowser(async browser => {
+    const page = new ButtonDropdownPage('ButtonDropdown1', browser);
+    await browser.url('#/light/button-dropdown/simple');
+    await page.waitForVisible(page.getTrigger());
+    await page.openDropdown();
+    const focusedElement = await browser.execute(function () {
+      return document.activeElement!.textContent;
+    });
+
+    const firstTabbableMenuItem = await browser.execute(
+      () => document.querySelector(`li [tabindex="-1"]`)!.textContent
```

Comment:
I think it is better to avoid using the same implementation in the test and in the code. It is better to simply copy here the actual item's text content from the test page.